addons/l10n_de/migrations/2.0/pre-migrate.py

```diff
@@ -0,0 +1,17 @@
+from odoo.upgrade import util
```

Comment:
This doesn't work for local upgrade scripts. It won't work for clients trying to upgrade their modules.

Reply:
Hi, this is my first time working on an upgrade script, can you explain a bit more why this won't work and what should be done instead?

Reply:
The `util` package is not available here. It is only available if you start Odoo with `--upgrade-path=...` **and** you have the utils package.

Reply:
Oh okay. In my test I did start it with `-u all --upgrade-path=...` so that's why it worked for me then. Is there a way to use the util package at all? Or should I write a query instead?